gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/CoopLockFsck.java

```diff
@@ -31,28 +32,34 @@
  * FSCK tool to recover failed directory mutations guarded by GCS Connector Cooperative Locking
  * feature.
  *
- * <p>Usage: <code>
- *   hadoop jar /usr/lib/hadoop/lib/gcs-connector.jar
- *       com.google.cloud.hadoop.fs.gcs.CoopLockFsck --rollForward gs://my-bucket
- * </code>
+ * <p>Usage:
+ *
+ * <pre>{@code
+ * hadoop jar /usr/lib/hadoop/lib/gcs-connector.jar  com.google.cloud.hadoop.fs.gcs.CoopLockFsck \
+ *     --rollForward gs://my-bucket <operation-id>
```

Comment:
Consider adding a shell script wrapper utility for this.

Reply:
Ack, it could be tricky though because we don't distribute any scripts with GCS connector and GCS connector jar path/location differs from environment to environment.